tinkerpop3/src/test/java/io/shiftleft/overflowdb/storage/GraphSaveRestoreTest.java

```diff
@@ -30,7 +30,7 @@ public void greenField() throws IOException {
     final Long vertex0Id;
     final Long vertex1Id;
     // create graph and store in specified location
-    try (OdbGraph graph = newGratefulDeadGraph(overflowDb, false)) {
+    try (OdbGraph graph = newGratefulDeadGraph(overflowDb, true)) {
```

Comment:
why would you enable the overflow for these tests?

Reply:
they did not pass because when reference manager does not collect node refs it did not save anything on close

Reply:
ah i see, ok then as discussed try to separate the two so that this will work without overflow enabled